### DISCUSSION TITLE
Does a typical SDXL model consume 13.6GB of VRAM when loaded?

### DISCUSSION THREAD

**Discussion by HubKing** [vladmandic/sdnext#2923] [Q&A] [answered]
For example, if I download [this 6.62GB SDXL model](https://civitai.com/models/133005/juggernaut-xl?modelVersionId=357609) or any other popular models and load it, the VRAM usage goes near 14GB and a high memory utilisation warning is printed in the terminal. I mean 13.6GB just when the model is loaded, not when generating an image. 

Is this normal? I don't remember exactly but I think the VRAM usage was not this high when I tried it months ago. I tried checking the two vram saving options (low/medium something), but it made no difference. Restarting the PC had no difference.

Intel Arc A770, 16GB VRAM, Linux.

**Comment by Disty0** [comment, marked as answer]
Don't use --no-half / FP32. Select BF16 from compute settings and restart.
Also disable ipex optimize model if you use Loras.

**Reply by HubKing** [reply]
Damn. This reduced the VRAM usage to 0.58GB. Just 16 and 32 fp (whatever that is) difference is this much VRAM difference?

I unchecked --no-half, --no-half-vae, changed Device precision type/preferred model variant/preferred VAE variant from fp32 to fp16,

**Reply by Disty0** [reply]
FP32 (32 Bit) is 2 times bigger and slower than BF16 / FP16 (16 Bit).
Also Intel ARC is faster with BF16 than FP16.
